.travis/push.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "info@tango-controls.org"
+  git config --global user.name "Tango CI"
+  git remote rm origin
+  git remote add origin https://tango-ci:$GITHUB_TOKEN@github.com/tango-controls/JTango.git
+}
+
+commit_website_files() {
+  git add . pom.xml
+  git commit --message "[Tango CI] Next development iteration"
+}
+
+upload_files() {
+  git push --quiet origin HEAD:issue-68
```

Comment:
@gwen-soleil this must be changed to 

```
git push --quiet origin HEAD:jtango-9-lts
```

before merge! Otherwise updated versions will be committed into this branch...

Maybe there is a better solution?!